samples/host_verify/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ cmake_minimum_required(VERSION 3.11)
 project("Remote Host-side Enclave Verification Sample" LANGUAGES C CXX)
 
 if (NOT BUILD_HOSTVERIFY_INLINE)
+  set(SKIP_OE_SGX_CHECK 1)
```

Comment:
How is SKIP_OE_SGX_CHECK used?

Reply:
If we build the sample after installing the host-verify package then it will search for SGX settings, which openenclave-config.cmake suggests, while the sample is supposed to run regardless of the settings. Therefore a variable is set up to skip if necessary.

Reply:
See the comment above.